spec/models/file_set_spec.rb

```diff
@@ -249,10 +249,6 @@
       expect(subject.read_groups).to eq ['group-6', 'group-7']
     end
 
-    it 'has read groups string accessor' do
```

Comment:
OK, basically all of my questions for this PR are about deletions related to permissions.